### DISCUSSION TITLE
Maintenance: Usage will be delayed for the next two days.

### DISCUSSION THREAD

**Discussion by madpilot** [amberelectric/public-api#121] [Announcements]
Just a quick heads up that we are doing some major technical maintenance work over the next few days - we are completing the migration to our own energy licence!

As a result, usage via the app and the API will be delayed for the next two or three days.

Apologies for any inconvenience caused and we’ll let you know as soon as possible when everything is back to normal.

**Comment by asanchezgiraldo** [comment]
@madpilot do you know when the API will be available again?

This post is 6 days old I am not able to see my data since Thursday now.

**Comment by madpilot** [comment]
No - we are working with the networks to resolve the issues, but I don't have an ETA at the moment. Sorry about that.

**Comment by madpilot** [comment]
I have it on good authority that the data feeds are all back. If you _still_ aren't seeing usage, contact info@amber.com.au and we can manually trigger a fetch for you.
